src/trixi.c

```diff
@@ -13,47 +13,105 @@ static jl_value_t* checked_eval_string(const char* code, const char* func, const
 
 
 
-void trixi_initialize(MPI_Fint* comm) {
-
-    // Convert Fortran communicator type to C
-    MPI_Comm comm_c = MPI_Comm_f2c(*comm);
+/** Initialize Julia runtime environment
+ *
+ *  \todo Path is still hard-coded
+ */
+void trixi_initialize() {
 
     // Init Julia
     jl_init();
 
-    // @todo
     // Activate julia environment
-    // checked_eval_string("using Pkg; Pkg.activate(\".\"); Pkg.status()", LOC);
+    checked_eval_string("using Pkg; Pkg.activate(\"../../LibTrixi.jl\"); Pkg.status()", LOC);
+
+    // Load LibTrixi module
+    checked_eval_string("using LibTrixi;", LOC);
+    checked_eval_string("println(\"Module LibTrixi.jl loaded\")", LOC);
+}
 
-    // Test Julia
-    checked_eval_string("println(\"libtrixi: Hi, I'm julia!\")", LOC);
 
-    // @todo
-    // Load Trixi
-    // checked_eval_string("using Trixi; trixi_include(default_example())", LOC);
+/** Setup Trixi simulation
+ *
+ *  \param[in]  elixir  path to file containing Trixi setup
+ *
+ *  \return handle (integer) to Trixi simulation instance
+ */
+int trixi_setup_simulation(const char * elixir) {
+
+    // Get function pointer
+    int (*trixi_setup_simulation_c)(const char *) = jl_unbox_voidpointer( checked_eval_string("trixi_setup_simulation_cfptr()", LOC) ) ;
+
+    // Call function
+    return trixi_setup_simulation_c( elixir );
```

Comment:
```suggestion
int trixi_setup_simulation(const char * libelixir) {

    // Get function pointer
    int (*trixi_setup_simulation_c)(const char *) = jl_unbox_voidpointer( checked_eval_string("trixi_setup_simulation_cfptr()", LOC) ) ;

    // Call function
    return trixi_setup_simulation_c( libelixir );
```
I'd call it `libelixir` (or we can come up with another name) to distinguish it from a regular elixir